internal/cli/iam/projects/settings/describe.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package settings
+
+import (
+	"context"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store store.ProjectSettingsDescriber
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var describeTemplate = `COLLECT DATABASE SPECIFICS STATISTICS ENABLED	DATA EXPLORER ENABLED	PERFORMANCE ADVISOR ENABLED	REALTIME PERFORMANCE PANEL ENABLED	SCHEMA ADVISOR ENABLED
```

Comment:
double-checking: will we display all the fields here?

Reply:
@blva I heard that it's good. Maybe @fmenezes will be able to confirm?

Reply:
@blva I don't follow the question, sorry.

Reply:
I think that is ok, the fields are all boolean, on a list we might choose the most important fields and display only those instead, in this case I can't tell which ones should be displayed and which ones are ok to be hidden. 🤷

Reply:
fair :D